tensorflow/python/compiler/tensorrt/__init__.py

```diff
@@ -15,5 +15,65 @@
 """Exposes the python wrapper for TensorRT graph transforms."""
 
 # pylint: disable=unused-import,line-too-long
+from tensorflow.compiler.tf2tensorrt import _pywrap_py_utils
 from tensorflow.python.compiler.tensorrt import trt_convert as trt
+from tensorflow.python.compiler.tensorrt import utils as trt_utils
+from tensorflow.python.platform import tf_logging as logging
 # pylint: enable=unused-import,line-too-long
+
+
+if not _pywrap_py_utils.is_tensorrt_enabled():
+  logging.error(
+      "Tensorflow needs to be built with TensorRT support enabled to allow "
+      "TF-TRT to operate.")
+
+  raise RuntimeError("Tensorflow has not been built with TensorRT support.")
+
+else:
```

Comment:
There is no need to use "else" if the if-branch will stop with an error. This allows us to reduce the indentation of the code after here.

Reply:
Fixed